docs/upgrading.md

```diff
@@ -244,7 +244,7 @@ You can opt-out of this new default by adding `excludeDefaultMomentLocales: fals
 
 ### Update usage of `router.events`
 
-In case you're accessing `router.events` during rendering, in Next.js 11 `router.events` is no longer provided during pre-rendering. Ensure you're accessing `router.events` in `useEffect`:
+In case you're accessing `router.events` during rendering, in Next.js 11 `router.events` are no longer provided during pre-rendering. Ensure you're accessing `router.events` in `useEffect`:
```

Comment:
```suggestion
In case you're accessing `router.events` during rendering, in Next.js 11 `router.events` is no longer provided during pre-rendering. Ensure you're accessing `router.events` in `useEffect`:
```

`router.events` is a single property, even if "events" is plural.